package.json

```diff
@@ -40,7 +40,7 @@
     "effective:circle:config": "circleci config process circle.yml | sed /^#/d",
```

Comment:
The 'clean-deps' command here at root is still using `rm`

Reply:
Yeah! I didn't changed bc it was changed 2 years ago and seems not to be an issue - but may worth it changing to be consistent across all the packages

<img width="766" alt="image" src="https://user-images.githubusercontent.com/15656860/161167834-38e428ba-f70d-4c72-9ee0-f8f42ec7d7c1.png">